changelog/_unreleased/2020-09-17-fix-listingpage-pagination-arrows-always-visible.md

```diff
@@ -0,0 +1,9 @@
+---
+title:          Hide unsupported arrows on pagination view
+issue:          NEXT-9023
+author:         Michał Jędraszczyk
+author_email:   m.jedrasz@gmail.com
+author_github:  @art4webs
+---
+# Storefront
+*  Hide prev arrows on first page and next arrows on last page in `pagination.html.twig`
```

Comment:
Those changelog entries **have to** start with either `Created`,  `Changed`, `Removed` or `Deprecated`